src/modules/dashboard/sagas/domains.ts

```diff
@@ -143,8 +140,50 @@ function* domainEdit({
   return null;
 }
 
+function* moveFundsBetweenPots({
+  payload: { colonyAddress, fromDomain, toDomain, amount, tokenAddress },
+  meta,
+}: Action<ActionTypes.MOVE_FUNDS_BETWEEN_POTS>) {
+  let txChannel;
+  try {
+    txChannel = yield call(getTxChannel, meta.id);
+    const [fromPot, toPot] = yield all([
```

Comment:
These are actually the domains though right? It's an object like `{ potId, skillId }`

Reply:
Yes! Thank you, I'll adjust!